app/views/users/show.html.erb

```diff
@@ -0,0 +1,45 @@
+<h1 class="text-center text-white bg-info rounded mt-1">This is <%= "#{@user.name}" %> personal page</h1>
+
+
+<hr>
+<div class="row">
+	<div class="col-md-6">
+		<% if @user.events.any? %>
+			<h3 class="text-center">Attended (<%= @user.events.count %>) events</h3>	
+			<ul class="media-list jumbotron">
+				<%= render @events %>
+			</ul>
+			<%= will_paginate @events %>
+		<% else %>
+			<h3>No Events Found</h3>
+		<% end %>
+	</div>
+
+	<div class="col-md-6">
+		<% if @created.any? %>
+			<h3 class="text-center">Created events</h3>
+			<% @created.select {|e| e.user_id == @user.id}.each do |event| %>
```

Comment:
- [ ] You shouldn't have any kind of logic on your views, the created events by a given user should be accessible with your created associations in the models and instances in the controller actions.